src/components/integrated/OakQuizHint/OakQuizHint.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { MouseEventHandler, ReactNode, useState } from "react";
+
+import { OakTertiaryButton, OakTooltip } from "@/components/ui";
+
+export type OakQuizHintProps = {
+  /**
+   * Some content to give as a hint to answer a question
+   */
+  hint: ReactNode;
+};
+
+/**
+ * Presents a button which will show a hint when clicked
+ */
+export const OakQuizHint = ({ hint }: OakQuizHintProps) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const handleClick: MouseEventHandler = () => {
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <OakTooltip tooltip={hint} isOpen={isOpen}>
```

Comment:
Should this be a tooltip or a modal that looks like a tooltip? 🦆 

It isn't triggered on hover, only on click so perhaps it should be viewed as `role="modal"`  for AT?